src/Board.py

```diff
@@ -35,3 +36,243 @@ def __init__(self, style="default", size=5) -> None:
                 raise ValueError("There is no default board for that number of players")
         else:
             raise ValueError("Un-recognised map style")
+
+    def get(self, x, y):
+        return self.data[y][x]
+
+    def getDirectlyAdj(self, start=(0, 0), TerainFilter=None) -> list:
```

Comment:
Could you please use `snake_case` for variable and function names? No particular reason aside from that's just the standard practice in Python. `CamelCase` for classes though